lib/browser.js

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+import historyPolyfill from 'html5-history-api';
```

Comment:
Ah, I think I see what you're saying.  Just to clarify and make sure we're on the same page, you want each individual project to require in the polyfill (if they need it) vs. tarantino doing it by default?  

And then inside tarantino, it would either use the polyfill or default to the `window.history.replaceState` or something of that sort correct?

eg. 

```
replaceHash: function(s) {
  if (window.historyPolyfill) {
    window.historyPolyfill.replaceState({}, document.title, s);
  } else {
    window.history.replaceState({}, document.title, s);
  }

  this.fire();

  return this;
}
```

Am I understanding this correctly? If I am, the only caveat is that we need to provide explicit instructions on how to insert the polyfill in the global namespace.  

eg. 
```
import historyPolyfill from 'html5-history-api;

// will not work and cause null pointer
window.polyfill = historyPolyfill;
```

Thoughts?